src/System.Management.Automation/engine/CommandDiscovery.cs

```diff
@@ -1281,27 +1281,6 @@ internal void UnregisterLookupCommandInfoAction(string currentAction, string com
         private HashSet<string> _activeCommandNotFound = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Command Discovery tests.Native command discovery.Error occurred in Context block
```powershell
Cannot process argument transformation on parameter 'Skip'. Cannot convert value "System.String" to type "System.Management.Automation.SwitchParameter". Boolean parameters accept only Boolean values and numbers, such as $True, $False, 1 or 0.
At /Users/vsts/agent/2.158.0/work/1/s/test/powershell/engine/Basic/CommandDiscovery.Tests.ps1:226 char:76
```